### PR TITLE
[Technical Support] LPS-69112 Calendar portlet styling is broken when displayed in a layout column smaller than the full page

### PR DESCRIPTION
**Pull request by dustinryerson**
/cc @brandizzi

Hey @jonmak08,

This is an update for https://issues.liferay.com/browse/LPS-69112. I've added screenshots to the ticket for a visual example of what the portlet looked like in smaller layout columns before and after the fix.

Below I'm adding some line comments to specify which parts of the code fix which layout columns. If you'd like me to divide them up into separate commits, let me know and I can resend.

Please let me know if you have any questions.

Thanks!

### CODE REVIEW

File: modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -681,4 +681,163 @@
 			font-size: 14px;
 		}
 	}
+}
+
+.col-md-4 {
```
Comment (dustinryerson) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss]: This fixes Calendar portlet styling when in a 30 column. After discussing this with @brandizzi, we thought styling it like the full page mobile calendar would be the best solution.

File: modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss [hunk 2]
```diff
@@ -681,4 +681,163 @@
...
+		}
+
+		.simple-color-picker-item {
+			height: 18px;
+			width: 18px;
+		}
+	}
+}
+
+.col-md-6, .col-md-8 {
```
Comment (dustinryerson) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss]: `col-md-6` fixes the styling in a 50 column, and `col-md-8` fixes stying in a 70 column.

**Comment by jonmak08** [comment]
Hey @dustinryerson,

I checked the changes for the most part everything is good.  There is one use case which is still causing issue thought I think.  If you create a page with a 70/30 layout or a 50/50 layout and you place the calendar portlet on the page and shrink the browser window, the calendar portlet sections should take up 100% of the page width correct?

This is what I see with your changes applied because your specificity is override the bootstrap defaults when the browser max width is changed to 100%.

70/30 layout:


50/50 layout:


Please let me know if you have any questions. Thanks!

**Comment by jonmak08** [comment]
I think if we had to add those bootstrap grid classes to the calendar's main.css, we would have to add all the media queries to the calendar's main.css as well to account for the mobile views.  I am not sure if there is another way/better way around this since duplicating all those CSS classes and media queries would not be ideal.

**Comment by jonmak08** [comment]
Closing this pull since we discussed the changes to be made.

**Comment by dustinryerson** [comment]
New pull: https://github.com/jonmak08/liferay-portal/pull/1632

**Pull request closed without merging**